### PR TITLE
samples: bluetooth: Add configuration for Thingy53 in mesh light switch sample

### PR DESCRIPTION
**Pull request by emob-nordic**
Jira: NCSDK-10308.

Kconfig warnings during compilation discussed and accepted by @trond-snekvik .

### CODE REVIEW

**Review by wiba-nordic — CHANGES_REQUESTED** [review]
Just a little improvement to one sentence, otherwise looks good.

File: samples/bluetooth/mesh/light_switch/README.rst [hunk 1]
```diff
@@ -44,8 +48,8 @@ This mobile application is also used to configure key bindings, and publication
 The Generic OnOff Client model is used for manipulating the Generic OnOff state associated with the Generic OnOff Server model.
 The light switch sample implements the Generic OnOff Client model.
 
-The sample has four buttons to control the state of LED 1 on servers (implemented by the :ref:`bluetooth_mesh_light` sample).
-Four instances of the Generic OnOff Client model are instantiated in the light switch sample, one for each button on the development kit that is used.
+The sample instantiates up to four instances of the Generic OnOff Client model to control the state of LEDs on servers (implemented by the :ref:`bluetooth_mesh_light` sample).
+Each instance of the Generic OnOff Client model are instantiated in the light switch sample, one for each button on the development kit that is available.
```
[the remark this answers is not in the document]
Comment (emob-nordic) [reply samples/bluetooth/mesh/light_switch/README.rst]: Done, please have a look once again if possible.

**Review by wiba-nordic — APPROVED** [review]
That looks good.

**Review by pdunaj — APPROVED** [review]

File: samples/bluetooth/mesh/light_switch/boards/thingy53_nrf5340_cpuapp.conf [hunk 2]
```diff
@@ -0,0 +1,71 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+################################################################################
+# Application overlay - Thingy:53 DFU
+
+CONFIG_MCUBOOT_IMAGE_VERSION="1.1.1+1"
```
[the remark this answers is not in the document]
Comment (emob-nordic) [reply samples/bluetooth/mesh/light_switch/boards/thingy53_nrf5340_cpuapp.conf:9]: Leftover after testing updates, will remove. Thanks!

**Pull request merged**